lib/delayed_resque/performable_method.rb

```diff
@@ -22,6 +24,16 @@ def display_name
       end
     end
 
+    def unique_job_id
+      return unless @options[:unique]
+
+      @unique_job_id ||= self.class.generate_unique_job_id(queue: queue)
+    end
```

Comment:
is there a reason this is not in the module? feels like it should be for some reason. maybe just the name?

Reply:
Good catch! It was a simple oversight on my part - I initially put the unique jobs implementation directly in `DelayedResque::PerformableMethod`, and then extracted it into the module later. I just missed this method. You're right, that is where it belongs.

Reply:
makes my tracked jobs code look bad... ;)